sweetest/sweetest-base/src/main/java/com/mysugr/sweetest/internal/TestContext.kt

```diff
@@ -0,0 +1,3 @@
+package com.mysugr.sweetest.internal
+
+interface TestContext
```

Comment:
What benefit do marker interfaces without any properties or functions provide here? 🤔

Reply:
it's not up-to-date after #41, because then there will be the `TestContext` that will hold `TestContextElement`s.

but in general, these empty interfaces are here because `sweetest-core` needs to manage these types (`TestContext`, `Stept`, ...) but does or did not know the concrete type which is only known to `sweetest-api`.